z80count/z80count.py

```diff
@@ -93,8 +182,15 @@ def parse_command_line():
                         help="Include subtotal")
     parser.add_argument('-n', dest='no_update', action='store_true',
                         help="Do not update existing count if available")
-    parser.add_argument('-t', dest='tabstop', type=int,
-                        help="Number of tabs for new comments", default=2)
+    parser.add_argument('-t', dest='tab_width', type=int,
+                        help="Number of spaces for each tab", default=8)
+    parser.add_argument('--use-spaces', dest='use_spaces', action='store_true',
+                        help="Use spaces to align newly added comments.", default=True)
```

Comment:
Please remove the period from the help strings for consistency.

Reply:
done